sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/export/IntervalMetricReader.java

```diff
@@ -141,8 +140,9 @@ CompletableResultCode doRun() {
                 flushResult.succeed();
                 exportAvailable.set(true);
               });
-        } catch (RuntimeException e) {
-          logger.log(Level.WARNING, "Exporter threw an Exception", e);
+        } catch (Throwable t) {
+          exportAvailable.set(true);
+          logger.log(Level.WARNING, "Exporter threw an Exception", t);
```

Comment:
I suspect this will end up being too noisy for someone, but we can address the "don't log so much" issue separately. See #3013